docs/operational_guide/repairs.md

```diff
@@ -0,0 +1,34 @@
+# Background Repairs (beta)
+
+## Overview
+
+An M3DB cluster can be configured to passively repair itself in the background. If the background repairs feature is enabled, then M3DB nodes will passively scan each other's metadata and when a mismatch is detected the nodes will perform a repair such that each node in the cluster eventually settles on a consistent view of the data.
```

Comment:
nit: "passively" is a little redundant here - if something is doing work in the background, arguably that work is "passive"